content/blog/future_events.md

```diff
@@ -0,0 +1,26 @@
++++
```

Comment:
I would suggest moving the content to content/about as opposed to blogs.

I'm not sure blog post is a good fit since we will want to continually update this.

I would also suggest having a single page and not distinguishing between future and past events but just listing events in reverse chronological order. 

This way we won't have to update the page every time an event actually happens; so it will just reduce the toil of keeping it up to date.

Reply:
This would mean that a top level "Content" page needs to be created, right? Currently, there is none.

Reply:
@jlewi The blog page currently has the title "Blogs, Talks, and Videos". We should probably change that to "Blogs and Events" and under that an Events page. "about" does not seem to be an appropriate title.

Reply:
@jlewi Got your comment. My fork was outdated :-) A lot has changed :-)